### PR TITLE
initial patch for copying assignment params for acdc

### LINKED ISSUE #7157: ReqMgr2: ACDCs do not inherit assignment information
you have to fill the form from scratch. I'm temporarily placing a screenshot here to make it clear:
https://www.dropbox.com/s/95wes2ujzrdhhyq/assignment.png?dl=0

### PR DESCRIPTION
**Pull request by ticoann**
fixes #7157

### CODE REVIEW

**Comment by ticoann** [comment]
@amaltaro, Alan, this patch still have problem. I made a request for you to review which parameters needs to be inherited. (site white list  - I need to change to get information from acdc)

File: src/couchapps/WMStats/_attachments/js/ViewModels/WMStats.ViewModel.js [hunk 1]
```diff
@@ -433,6 +433,37 @@ WMStats.ViewModel = (function (){
             summary.SoftwareVersions = requestInfo.SoftwareVersions;
             summary.CMSSWVersion = requestInfo.SoftwareVersions[0];
             summary.Campaign = requestInfo.Campaign;
+            summary.Team = requestInfo.Teams[0];
+            summary.Teams = requestInfo.Teams;
```
Comment (amaltaro) [inline src/couchapps/WMStats/_attachments/js/ViewModels/WMStats.ViewModel.js]: I think only `Team` should be enough.

**Comment by amaltaro** [comment]
Ok, besides the `team` comment, I still have these general comments:

Parameters that can be dictionary and tricky to retrieve:
summary.AcquisitionEra
summary.MaxRSS
summary.ProcessingString
summary.ProcessingVersion
summary.MaxVSize
are they considered in this patch?

{Possible} not needed:
summary.GlobalTag
summary.ScramArch
summary.TrustSitelists
summary.TrustPUSitelists
and can be removed (probably they should)

**Comment by ticoann** [comment]
duplicate

**Pull request closed without merging**